### PR TITLE
Added completions for symbols from unimported pkg if there are multiple matches for pkg name

### LINKED ISSUE #1884: No completions for symbols from unimported pkg if there are multiple matches for pkg name
`fmt.` gives completions for all the symbols from the fmt package
`template.` doesn't do the same.

This is because we cannot determine which of the `text/template` and `html/template` should be used.

Proposed fix:
- When encountering `template.` provide completions for all the matching pkgs
- Once user chooses the package, the import should be added for that package, and completions should be triggered again.

This is possible to do because we can run commands as part of the completion. But since we can run one 1 command for a completion item, we will have to create a new command that internally runs `go.import.add` command followed by triggering suggestions again  with `editor.action.suggest` command

### PR DESCRIPTION
**Pull request by shreyu86**
Fixes #1884

### CODE REVIEW

File: src/goSuggest.ts [hunk 1]
```diff
@@ -131,11 +131,28 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 								// add additionalTextEdits to do the same in the actual document in the editor
 								// We use additionalTextEdits instead of command so that 'useCodeSnippetsOnFunctionSuggest' feature continues to work
 								newsuggestions.forEach(item => {
-									item.additionalTextEdits = getTextEditForAddImport(pkgPath);
+									item.additionalTextEdits = getTextEditForAddImport(pkgPath[0]);
```
Comment (ramya-rao-a) [inline src/goSuggest.ts]: Shouldnt we use `pkgPath[0]` at line 124 as well?

**Comment by shreyu86** [comment]
Thanks, @ramya-rao-a I tested the new change. How do I go about updating the unit test? Can I hit tab on the first suggestion to trigger subsequent suggestion and then check for those?

**Comment by ramya-rao-a** [comment]
Or you can run the command `acceptSelectedSuggestion`

**Comment by shreyu86** [comment]
@ramya-rao-a I will update the tests and have this PR up for review soon, sorry I was out of office.

**Comment by ramya-rao-a** [comment]
No problem, take your time. I was out last week on vacation as well :)

**Comment by shreyu86** [comment]
@ramya-rao-a I attempted to refactor the tests but was not successful in `acceptSelectedSuggestion` can you please provide some samples/pointers?

**Comment by ramya-rao-a** [comment]
- After `showTextDocument`, set `editor.selection` to the right position where we want suggestions to be triggered
- Then use `vscode.commands.executeCommand('editor.action.triggerSuggest')` followed by `vscode.commands.executeCommand('acceptSelectedSuggestion')`
- Then check the document contents

Note: These are just ideas, I havent tried them out myself :)

**Comment by shreyu86** [comment]
@ramya-rao-a I tried couple of options ^^ and I was not able to get the test refactored, can you please help?

**Comment by ramya-rao-a** [comment]
The problem is that the suggest widget is not completely rendered before the accept suggestion command is executed.

Never mind, I think we can skip that.

I pushed a commit for some extra checks in your test case. This is good to go.

Thanks again!

**Comment by shreyu86** [comment]
Thanks a lot for help on this! Really appreciate it.

**Comment by shreyu86** [comment]
Thanks a lot @ramya-rao-a

**Comment by ramya-rao-a** [comment]
No problem at all

**Pull request merged**